launch/test/test_module_level_launch.py

```diff
@@ -1,63 +0,0 @@
-# Copyright 2015 Open Source Robotics Foundation, Inc.
```

Comment:
@dirk-thomas I couldn't figure out what these tests were trying to accomplish, so I couldn't determine how I might refactor it to still test this without the async launcher. Do you have any insight? My uneducated guess is that it can just be removed.